README.md

```diff
@@ -37,6 +40,16 @@ Right-click the folder named **ui** and click **Export**. This will download a f
 
 ![](/documentation/download-content.gif)
 
+### Optional: Download the content of the pivot folder
+
+:warning: If there are calculated measures to be migrated from ActiveUI-4 to the ActiveUI-5 content servers, this file is required. If this is not the case, this file can be disregarded.
+
+The pivot folder can be downloaded as a JSON file from the ActiveViam Content Manager. If you are used to accessing ActiveUI at `https://my-server.com/ui/index.html`, then you should be able to access your Content Manager at `https://my-server.com/content/ui/index.html`.
```

Comment:
```suggestion
The pivot folder can be downloaded as a JSON file from the atoti+ Admin UI. If you are used to accessing ActiveUI at `https://my-server.com/ui/index.html`, then you should be able to access the Admin UI at `https://my-server.com/admin/ui/index.html`.
```

Reply:
I don't think we need to mention the "if you are used to..." part twice  since we mention it right above.
However, let's adapt the first one that it is mentioned with more up to date doc.